apps/courses/tests/test_wizards.py

```diff
@@ -0,0 +1,225 @@
+"""
+Test suite for the wizard creating a new Course page
+"""
+from django.core.urlresolvers import reverse
+
+from cms.api import create_page
+from cms.models import Page
+from cms.test_utils.testcases import CMSTestCase
+
+from apps.core.factories import UserFactory
+
+from ..cms_wizards import CourseWizardForm
+from ..factories import CourseFactory
+from ..models import Course, CoursePage, COURSES_PAGE_REVERSE_ID
+
+
+class CourseCMSWizardTestCase(CMSTestCase):
+    """Course tests"""
```

Comment:
more like : Testing the wizard that is used to create new course pages from the CMS"

Reply:
ok